### ISSUE TITLE
v1.2.0 configure error

### ISSUE THREAD

**Issue by dtmistry** [fluent/fluent-plugin-prometheus#66]
I've a below simple fluentd config - 

```xml
<system>
    @log_level debug
</system>

# Listen to events on Unix domain socket
<source>
  @type unix
  path /var/run/events.sock
  @log_level debug
</source>

# Monitor retries and emit events from various plugins
<source>
  @type prometheus_output_monitor
  interval 10
  <labels>
    hostname ${hostname}
  </labels>
  @id output_monitor
</source>

# Enable prometheus metrics
<source>
  @type prometheus
  bind 0.0.0.0
  port 24231
  metrics_path /prometheus/metrics
</source>

# Filter plugin to count incoming events
<filter events.*>
  @type prometheus
  @id prometheus_filter_plugin
  <metric>
    name fluentd_events_input_num_records_total
    type counter
    desc The total number of incoming records
    <labels>
      tag ${tag}
      hostname ${hostname}
    </labels>
  </metric>
</filter>

<match events.*>
  @type copy
  <store>
    # count outgoing events
    @id prometheus_output_plugin
    @type prometheus
    <metric>
      name fluentd_events_output_num_records_total
      type counter
      desc The total number of outgoing events
      <labels>
        tag ${tag}
        hostname ${hostname}
      </labels>
    </metric>
  </store>
  <store>
    @type stdout
  </store>
</match>
```

And I get the below error when starting fluentd v0.12 - 

```bash
/usr/lib/ruby/gems/2.4.0/gems/fluentd-1.2.6/lib/fluent/plugin/filter_record_transformer.rb:230:in `expand': undefined method `gsub' for #<Fluent::PluginHelper::RecordAccessor::Accessor:0x00007fc76fd95e70> (NoMethodError)
        from /usr/lib/ruby/gems/2.4.0/gems/fluent-plugin-prometheus-1.2.0/lib/fluent/plugin/in_prometheus_output_monitor.rb:45:in `block in configure'
        from /usr/lib/ruby/gems/2.4.0/gems/fluent-plugin-prometheus-1.2.0/lib/fluent/plugin/in_prometheus_output_monitor.rb:44:in `each'
        from /usr/lib/ruby/gems/2.4.0/gems/fluent-plugin-prometheus-1.2.0/lib/fluent/plugin/in_prometheus_output_monitor.rb:44:in `configure'
        from /usr/lib/ruby/gems/2.4.0/gems/fluentd-1.2.6/lib/fluent/plugin.rb:164:in `configure'
        from /usr/lib/ruby/gems/2.4.0/gems/fluentd-1.2.6/lib/fluent/root_agent.rb:282:in `add_source'
        from /usr/lib/ruby/gems/2.4.0/gems/fluentd-1.2.6/lib/fluent/root_agent.rb:122:in `block in configure'
        from /usr/lib/ruby/gems/2.4.0/gems/fluentd-1.2.6/lib/fluent/root_agent.rb:118:in `each'
        from /usr/lib/ruby/gems/2.4.0/gems/fluentd-1.2.6/lib/fluent/root_agent.rb:118:in `configure'
        from /usr/lib/ruby/gems/2.4.0/gems/fluentd-1.2.6/lib/fluent/engine.rb:131:in `configure'
        from /usr/lib/ruby/gems/2.4.0/gems/fluentd-1.2.6/lib/fluent/engine.rb:96:in `run_configure'
        from /usr/lib/ruby/gems/2.4.0/gems/fluentd-1.2.6/lib/fluent/supervisor.rb:795:in `run_configure'
        from /usr/lib/ruby/gems/2.4.0/gems/fluentd-1.2.6/lib/fluent/supervisor.rb:579:in `dry_run'
        from /usr/lib/ruby/gems/2.4.0/gems/fluentd-1.2.6/lib/fluent/supervisor.rb:597:in `supervise'
        from /usr/lib/ruby/gems/2.4.0/gems/fluentd-1.2.6/lib/fluent/supervisor.rb:502:in `run_supervisor'
        from /usr/lib/ruby/gems/2.4.0/gems/fluentd-1.2.6/lib/fluent/command/fluentd.rb:310:in `<top (required)>'
        from /usr/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from /usr/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from /usr/lib/ruby/gems/2.4.0/gems/fluentd-1.2.6/bin/fluentd:8:in `<top (required)>'
        from /usr/bin/fluentd:23:in `load'
        from /usr/bin/fluentd:23:in `<main>'
```

*Versions*
Fluentd - v0.12
fluent-plugin-prometheus - 1.2.0

**Comment by ferbar** [comment]
This change is propably breaking the label tag:

https://github.com/fluent/fluent-plugin-prometheus/commit/ceb7110c666894e4751bdc6355f7a3f4c97ecd0f#diff-94f973da7e7419c9e41608633bfac52b

**Comment by richm** [comment]
yep - broken - please fix asap

**Comment by kazegusuri** [comment]
Now I released v1.2.1. Please try the version and reopen if you still have a problem. Sorry for inconvenience.
